Formula/c/chapel.rb

```diff
@@ -59,20 +74,37 @@ def install
       CHPL_TARGET_MEM=jemalloc
       CHPL_TARGET_JEMALLOC=system
       CHPL_HWLOC=system
+      CHPL_LLVM=system
       CHPL_LLVM_CONFIG=#{llvm.opt_bin}/llvm-config
       CHPL_LLVM_GCC_PREFIX=none
+      CHPL_TARGET_CPU=native
     EOS
+    # NOTE: CHPL_TARGET_CPU=native could cause problems for users cross-compiling,
+    # should we only set this for CHPL_COMM=none?
 
     # Must be built from within CHPL_HOME to prevent build bugs.
     # https://github.com/Homebrew/legacy-homebrew/pull/35166
     cd libexec do
       system "./util/printchplenv", "--all"
-      with_env(CHPL_LLVM: "none") do
+      system "make"
```

Comment:
any docs for the build flow?

Reply:
Yes, see https://chapel-lang.org/docs/main/usingchapel/chplenv.html#chpl-llvm